tools/distpartitioning/dataset_utils.py

```diff
@@ -15,64 +18,90 @@ def get_dataset(input_dir, graph_name, rank, num_node_weights):
         graph name string
     rank : int
         rank of the current process
+    world_size : int
+        total number of process in the current execution
     num_node_weights : int
         integer indicating the no. of weights each node is attributed with
```

Comment:
this argument doesn't exist any more.

Reply:
Removed the 'num_node_weights' from the docstring. Added 'schema_map' to the docstring as it is used now in the multiple-file-format dataset.